### PR TITLE
Feature/company controller

### PR DESCRIPTION
**Pull request by olcplif**
- [x] Add Companies controller (admins side).
- [x] Add Companies controller (clients side).
- [x] Add routes (admins and clients side)
- [x] Add requests tests.

[https://trello.com/c/Eoy4ZTyk/32-create-companies-page](https://trello.com/c/Eoy4ZTyk/32-create-companies-page)

### CODE REVIEW

File: app/controllers/admins/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    end
+
+    def show; end
+
+    def new
+      @company = Company.new
+    end
+
+    def create
+      @company = Company.new(company_params)
```
Comment (mmrshk) [inline app/controllers/admins/companies_controller.rb:18]: add empty line

File: app/helpers/companies_helper.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CompaniesHelper
```
Comment (mmrshk) [inline app/helpers/companies_helper.rb:3]: delete, we are not using helper

are you using scaffolding? if yes, please, stop doing this

**Review by mmrshk — APPROVED** [review]
LGTM!

File: app/controllers/admins/companies_controller.rb (hunk 1 above)
Comment (olcplif) [reply app/controllers/admins/companies_controller.rb:18]: added

File: app/helpers/companies_helper.rb (hunk 2 above)
Comment (olcplif) [reply app/helpers/companies_helper.rb]: deleted

Comment (olcplif) [reply app/helpers/companies_helper.rb]: I used `rails g controller`

**Pull request merged**